src/lt/objs/plugins.cljs

```diff
@@ -494,7 +494,7 @@
    (search-input this)])
 
 (defui source-button [plugin]
```

Comment:
That's perfectly understandable. :smile:

Since I'm not sure if it would break customized themes, I am leaning towards leaving it as another issue (looks like issue #2290) and PR that would be merged in the future with along other potentially breaking changes we have planned.